src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import cn from 'classnames';
+import { FilterTypes } from '../../types/FilterTypes';
+import React from 'react';
+
+interface Props {
+  unfinishedTodoAmount: number;
+  setFilterInstructions: (filterType: FilterTypes) => void;
+  filterInstructions: FilterTypes;
+}
+
+export const Footer: React.FC<Props> = ({
+  unfinishedTodoAmount,
+  setFilterInstructions,
+  filterInstructions,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${unfinishedTodoAmount} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: filterInstructions === FilterTypes.All,
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilterInstructions(FilterTypes.All)}
+        >
+          {FilterTypes.All}
+        </a>
```

Comment:
You can convert your enum into an array with `Object.keys/values/entries` and map through it to generate links